components/CharacterInfo.js

```diff
@@ -0,0 +1,76 @@
+function SingleStat(props) {
+  return (
+    <div className="flex">
+      <div className="w-1/2 uppercase">{props.name}</div>
+      <div className="w-1/2 font-light">{props.value}</div>
+    </div>
+  )
+}
+
+function CharacterInfo(props) {
+  const character = props.character;
+  return (
+    <div className="flex p-6 background-repeat"
+      style={{
+        backgroundImage: "url('https://cdn.discordapp.com/attachments/877530614138560603/877587502142201926/unknown.png')",
+        }}>
+      <div>
+        <img 
+        className="flex-none w-auto p-2 px-12 border-2 border-white border-opacity-50 bg-white bg-opacity-20"
+
+        src={character.illustration} alt="NieR Re[in]carnation Launched !" />
+      </div>
+      <div className="flex-1 mx-2 px-4">
+        <span style={{
+          background: "lightgrey",
```

Comment:
Is there a specific reason we are using style objects instead of (Tailwind) classes ?

Reply:
nope, just old code